### PR TITLE
Credits and organising the folder.

### LINKED ISSUE #64: Give more credit to original first contributions repo
As raised by one of our open source challengers

### PR DESCRIPTION
**Pull request by guyandtheworld**
#### Credits (Adarsh)

The credits for the idea, and the content completely goes to first-contribution repository. We, at The Fast Tracked Programmer, absolutely loved their implementation of first-contributions project and decided to implement it for the participants of the Open-Source challenge.

We decided to add a small enhancement to the already existing idea. We decided to pair the first contributions with a kanban board within Github. Additionally, since our mission is to get people to make meaningful Open-Source Pull-Requests, we decided to use the markdown files people push onto the first-contributions folder to act as an online resume where they can document their contributions to Open-Source. We decided to erase the commit history so that the particular folder inside this repository would be easy to navigate and read through other people's contribution.

#### Credits (Adam)

Special thanks to [firstcontributions](https://github.com/firstcontributions) for inspiring us to build upon their incredible work. We both share a vision of wanting to help beginners to contribute to open source projects, and used their [first-contributions](https://github.com/firstcontributions/first-contributions) repository as a base.

We have not forked their repository or preserved commit history because we wanted to keep things as simple as possible for our community of aspiring contributors. For reference, the original MIT licence can be found in our repository [here](https://github.com/the-fast-tracked-programmer/first-contributions/blob/main/LICENSE).

#### Preview

https://github.com/guyandtheworld/first-contributions

#### Consequence

closes #64

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,10 +101,18 @@ Our mentors will then be notified of your pull request and either approve (and m
...
+
+## Credits
+
+The credits for the idea, and the content completely goes to [first-contribution](https://github.com/firstcontributions/first-contributions) repository. We, at The Fast Tracked Programmer, absolutely loved their implementation of first-contributions project and decided to implement it for the participants of the Open-Source challenge.
+
+We decided to add a small enhancement to the already existing idea. We decided to pair the first contributions with a kanban board within Github. Additionally, since our mission is to get people to make meaningful Open-Source Pull-Requests, we decided to use the markdown files people push onto the first-contributions folder to act as an online resume where they can document their contributions to Open-Source. We decided to erase the commit history so that the particular folder inside this repository would be easy to navigate and read through other people's contribution.
+
+Grateful for the team [firstcontributions](https://github.com/firstcontributions) for coming up with this idea and Open-Sourcing it.
+
+
```
Comment (bedrock-adam) [inline README.md:113]: extra line here :fire:

Comment (Subskii) [inline README.md:113]: :fire:  extra line here?

**Comment by Subskii** [comment]
Let's let our challengers move those files so that they can gain more experience.

Have opened PR's for them.

File: README.md [hunk 2]
```diff
@@ -101,10 +101,18 @@ Our mentors will then be notified of your pull request and either approve (and m
...
 ## Where to go from here?
 
 Congratulations! You just completed the standard _fork -> clone -> edit -> pull request_ workflow that you'll encounter often as a contributor!
 
-See our [list of projects](https://firstcontributions.github.io/#project-list) to help you continue your journey into open source.
-
```
Comment (Subskii) [inline README.md:109]: :+1:

**Comment by Subskii** [comment]
Have updated the pull request description with my take, based on yours.

What do you think think @guyandtheworld?

**Pull request merged**